File: Shange/CollectionViewCell.m
//
//  CollectionViewCell.m
//  missingkids
//
//  Created by Gal Blank on 9/23/15.
//  Copyright © 2015 Gal Blank. All rights reserved.
//

#import "CollectionViewCell.h"

@implementation CollectionViewCell

@synthesize label,imageview;

- (id)initWithFrame:(CGRect)frame {
    self = [super initWithFrame:frame];
    
    if (self) {
        imageview = [[UIImageView alloc] initWithFrame:self.frame];
        self.backgroundView = imageview;
        //[self addSubview:self.imageview];
        self.layer.borderWidth = 0.5;
        self.layer.borderColor = [UIColor grayColor].CGColor;
        //self.layer.cornerRadius = 10.0;
        self.label = [[InsetUILabel alloc] initWithFrame:CGRectMake(0, self.frame.size.height - 50, self.frame.size.width, 50)];
        self.autoresizesSubviews = YES;
        self.label.backgroundColor = [UIColor blackColor];
        self.label.alpha = 0.7;
        self.label.numberOfLines = 0;
        self.label.layer.masksToBounds = YES;
        //self.label.layer.cornerRadius = 10.0;
        self.label.autoresizingMask = (UIViewAutoresizingFlexibleWidth |
                                       UIViewAutoresizingFlexibleHeight);
        self.label.font = [UIFont fontWithName:@"HelveticaNeue-Bold" size:11];
        self.label.textColor = [UIColor whiteColor];
        self.label.textAlignment = NSTextAlignmentLeft;
        self.label.adjustsFontSizeToFitWidth = YES;
        [self addSubview:self.label];
    }
    return self;
}


@end

File: Shange/CollectionViewCell.swift
//
//  CollectionViewCell.swift
//  Shange
//
//  Created by Gal Blank on 10/28/15.
//  Copyright © 2015 Gal Blank. All rights reserved.
//

import UIKit

class CollectionViewCell: UICollectionViewCell {
    
    var textLabel: UILabel!
    var imageView: UIImageView!
    
    override init(frame: CGRect) {
        super.init(frame: frame)
        
        imageView = UIImageView(frame: CGRect(x: 0, y: 0, width: frame.size.width, height: frame.size.height))
        imageView.contentMode = UIViewContentMode.ScaleAspectFit
        contentView.addSubview(imageView)
        
        /*textLabel = UILabel(frame: CGRect(x: 0, y: imageView.frame.size.height, width: frame.size.width, height: frame.size.height/3))
        textLabel.font = UIFont.systemFontOfSize(UIFont.smallSystemFontSize())
        textLabel.textAlignment = .Center
        contentView.addSubview(textLabel)*/
    }

    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
}
